### DISCUSSION TITLE
Missing custom-conda-path.txt

### DISCUSSION THREAD

**Discussion by Koxae** [hlky/stable-diffusion-webui#865] [Q&A]
I run Anaconda Prompt, drag and drop webui-streamlit.cmd into this window, or run webui.cmd and get an error: 
"Cannot find custom-conda-path.txt file.
anaconda3/miniconda3 detected in C:\ProgramData\anaconda3"
There is no such file at this address. I don't know where I can get this file. Please help me!

**Comment by sanscolour** [comment]
No problem.  I had the same issue.  you are right, the file doesn't exist, you will have to make it.  This will be fast.

This error is happening because it expected to find conda in the default location.  I think that is \user\miniconda3\ or some such.  likely, you installed it in a non-default location.  Easy fix.

create a file in your Stable Diffusion root folder called "custom-conda-path.txt".  Open that file, place the path to the conda root in that file, save, and you are done.

**Reply by Koxae** [reply]
Thanks for the answer!
I did this, but it didn't help. I will try to somehow make the SD work. So far I haven't been able to do it. )
